cmd/mt-fakemetrics/cmd/parrot.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/schema"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	parrotOrgId          int
+	parrotPartitionCount int32
+	partitionMethod      schema.PartitionByMethod
+)
+
+func init() {
+	rootCmd.AddCommand(parrotCmd)
+	parrotCmd.Flags().IntVar(&parrotOrgId, "parrot-org-id", 1, "org id to publish parrot metrics to")
+	parrotCmd.Flags().Int32Var(&parrotPartitionCount, "parrot-partition-count", 128, "number of partitions to publish parrot metrics to")
+}
+
+var parrotCmd = &cobra.Command{
+	Use:   "parrot",
```

Comment:
initial card said `parrot-tests`, do we want to change to that?

Reply:
no. there is no need for the "-tests" suffix AFAICT